src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,119 @@
+.main {
+  display: grid;
+  justify-items: center;
+  justify-content: center;
+  grid-template-columns: repeat(4, 200px);
+  grid-template-rows: repeat(2, auto);
+  gap: 46px 48px;
+  padding: 50px 40px;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid $card-border-color;
+  transition: transform $transition-duration ease;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__img {
+    height: 134px;
+    margin: 32px 19px 0;
+  }
+
+  &__description {
+    color: $card-main-color;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-top: 40px;
+    margin-inline: 16px;
+    margin-bottom: 0;
+
+    &:hover {
+      color: $title-hover-color;
+    }
+  }
+
+  &__code {
+    color: $card-secondary-color;
+    font-size: 10px;
```

Comment:
In the `main.scss` file, the `font-size` rule is already set 10 px, so there is no need to duplicate this rule further.